samples/bluetooth/mesh/light/README.rst

```diff
@@ -98,35 +98,23 @@ Testing consists of provisioning the device and configuring it for communication
 Provisioning the device
 -----------------------
 
-The provisioning assigns an address range to the device, and adds it to the mesh network.
-Complete the following steps in the nRF Mesh app:
+.. |device name| replace:: :guilabel:`Mesh Light`
 
-1. Tap :guilabel:`Add node` to start scanning for unprovisioned mesh devices.
-#. Select the :guilabel:`Mesh Light` device to connect to it.
-#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
-#. When prompted, select the OOB method and follow the instructions in the app.
-
-Once the provisioning is complete, the app returns to the Network screen.
+.. include:: /includes/mesh_device_provisioning.txt
 
 Configuring models
 ------------------
 
-Complete the following steps in the nRF Mesh app to configure models:
+See :ref:`ug_bt_mesh_model_config_app` for details on how to configure the mesh models with the nRF Mesh mobile app.
 
-1. On the Network screen, tap the :guilabel:`Mesh Light` node.
-   Basic information about the mesh node and its configuration is displayed.
-#. In the mesh node view, expand the first element.
-   It contains the list of models in the first element of the node.
-#. Tap :guilabel:`Generic OnOff Server` to see the model's configuration.
-#. Bind the model to application keys to make it open for communication:
+Configure the Generic OnOff Client model on each element on the :guilabel:`Mesh Light` node:
 
-   1. Tap :guilabel:`BIND KEY` at the top of the screen.
-   #. Select :guilabel:`Application Key 1` from the list.
+* Bind the model to :guilabel:`Application Key 1`.
 
-   You are now able to control the first LED on the device by using the Generic On Off Controls in the model view.
-#. Tap :guilabel:`ON` to light up the first LED on the development kit.
+  You are now able to control the first LED on the device by using the Generic On Off Controls in the model view.
+* Tap :guilabel:`ON` to light up the first LED on the development kit.
 
-Repeat steps 3-5 for each of the elements on the node to enable controling each of the remaining three LEDs.
+Make sure to complete the configuration on each of the elements on the node to enable controling each of the remaining three LEDs.
 
```

Comment:
I believe controlling needs two L:s

Reply:
hahaha You are right, this one I totally went blind on :)